db/seeds.rb

```diff
@@ -1,6 +1,19 @@
 # Development seeds
+def generate_base_url(env_name)
```

Comment:
The envs created on dev, staging or pre-prod will not be able to correctly query the API so the actual base_url we're setting here doesn't matter too much for the time being, but I thought we should make it follow the same pattern anyway.  It might at some point become outdated if the url pattern changes.